qutebrowser/browser/webengine/darkmode.py

```diff
@@ -236,6 +248,22 @@ def copy_add_setting(self, setting: _Setting) -> '_Definition':
         new._settings = self._settings + (setting,)  # pylint: disable=protected-access
         return new
 
+    def copy_replace_setting(self, option: str, chromium_key: str) -> '_Definition':
+        """Get a new _Definition object with `old` replaced by `new`.
+
+        If `old` is not in the settings list, return the old _Definition object.
+        """
+        new = copy.deepcopy(self)
+
+        for setting in new._settings:  # pylint: disable=protected-access
+            if setting.option == option:
+                setting.chromium_key = chromium_key
+                break
+        else:
+            raise ValueError(f"Setting {option} not found in {self}")
+
+        return new
```

Comment:
I wonder if an early return would be cleaner here:
```suggestion
        for setting in new._settings:  # pylint: disable=protected-access
            if setting.option == option:
                setting.chromium_key = chromium_key
                return new

        raise ValueError(f"Setting {option} not found in {self}")
```

Reply:
We agreed on the early return approach (discussed on IRC).